### DISCUSSION TITLE
How to get debug messages on the movie/picture created?

### DISCUSSION THREAD

**Discussion by sroller** [Motion-Project/motionplus#109] [Q&A] [answered]
In Motion I remember I got some information on the left upper corner of the pictures about the status, like Precap, trigger, postcap etc. Is that still available in Motion+ and if the answer is yes then how can I activate it?

**Comment by Mr-Dave** [comment, marked as answer]
I think it should still be available by specifying a  `log_level` of debug.

**Reply by sroller** [reply]
That worked!
A minor comment: I had to scale up the text fonts (4x) because I'm running in 4K resolution. Now the lines overlap.
![image](https://github.com/Motion-Project/motionplus/assets/596136/f1c103c8-3ee7-4c10-b250-3a7629cb696e)
